operator/cmd/mesh/install.go

```diff
@@ -139,6 +141,22 @@ func InstallManifests(setOverlay []string, inFilenames []string, force bool, dry
 	if err != nil {
 		return err
 	}
+
+	serverVersion, err := clientset.Discovery().ServerVersion()
+	if err != nil {
+		return fmt.Errorf("error getting Kubernetes version: %w", err)
+	}
+	ok, err := k8sversion.CheckKubernetesVersion(serverVersion)
+	if err != nil {
+		return fmt.Errorf("error checking if Kubernetes version is supported: %w", err)
+	}
+	if !ok {
+		l.LogAndPrintf("\nThe Kubernetes version %s is not supported by Istio %s. The minimum supported Kubernetes version is %s.\n"+
+			"Proceeding with the installation, but you might experience problems. "+
```

Comment:
Why proceed? In K8s <=1.15, the Istio 1.7 CRDs cannot be installed. So proceeding will cause some problem.

I'd suggest default to abort. And if users really want to proceed, use --force.

Reply:
@howardjohn what do you think about this approach?

Reply:
I am fine with make it require force. I am not ok with not having a way to install on an unsupported version